aimmo-game/simulation/avatar/avatar_wrapper.py

```diff
@@ -11,6 +19,36 @@ def __init__(self, initial_location, player_id, worker_url, avatar_appearance):
         self.player_id = player_id
         self.avatar_appearance = avatar_appearance
         self.worker_url = worker_url
+        self.effects = set()
+        self.resistance = 0
+        self.attack_strength = 1
+
+    def take_turn(self, game_state, game_view):
+        action = self._get_action(game_view)
+        if action is not None:
+            action.apply(game_state, self)
+            LOGGER.debug("%s took %s" % (self.player_id, action))
+        effects_to_remove = set()
```

Comment:
`effects_to_remove = { effect for effect in self.effects if not effect.turn() }` might be more clear and more concise ?